### ISSUE TITLE
66% threshold rationale

### ISSUE THREAD

**Issue by Ge0rges** [cultivarium/MicrobeMod#3]
After looking over the preprint, I was left wondering how the 66% threshold was picked? My understanding is that `modkit` calculates and picks a suitable threshold on it's own, so I'm curious to know why this decision was made.

**Comment by Ge0rges** [comment]
It would be a useful supplement to present a benchmark with different thresholds, or a perhaps some logic where you let modkit calculate the threshold for different samples and pick the highest one?

**Comment by alexcritschristoph** [comment]
Yeah, I was thinking a bit more about implementing something like this today but, I think I have decided against it. My thinking:

1. Fortunately, I have found that motif calling is generally robust to parameters chosen within the 50%-90% range, which modkit generally chooses. So I don't expect this to have an enormous impact. 

2. While I'm pretty sure the modkit automatic cutoff choices are principled, I'm not sure how they interact with edge cases (either weird microbial genomes, low coverage, or something else). I wanted to avoid this possible sample-to-sample source of uncertainty. When running it I did encounter some cases where it seemed to choose a weirdly high cutoff for an individual genome, and it wasn't quite clear to me why.... 

3. Benchmarking this is tricky. Changing these cutoffs can sometimes have an effect on the motif sequence called - but how do we compare motifs? Edit distance? Something else? It's hard to think of a metric that completely captures how "right" a motif is. And my dataset is fairly limited to then test it at scale.

4. In general my advice here to everyone is to try out a few cutoffs and see what happens, comparing the motif output, the intermediate output from STREME, etc. For large systematic analyses of many genomes. Very curious though about cases where modkit's automatic thresholds are very different and strongly (and correctly) change the biological conclusions - I'm not sure yet what these cases may be.
